scripts/migrate-group.js

```diff
@@ -184,7 +184,7 @@ async function updatePolicyFiles({dryRun = true, policyFiles, oldBucket}) {
 
   console.group(`\nRemoving ${oldBucketArn} resources from local policy files`);
 
-  let updatedFiles = [];
+  const updatedFiles = [];
```

Comment:
> I guess eslint wasn't checking this file before?

Yeah, `npm run lint` is configured to [run ESLint on files ending with `.js` and `.jsx`](https://github.com/nextstrain/nextstrain.org/blob/adae78a29e34a38aba9bc91bf33d32ba0070976f/package.json#L16).

This was an unintentional side effect of cf0d1bf430c85081de06e5fc1acdd86a4e159a46, but I think it's good to lint these files anyways.

Reply:
Re: `let` vs. `const`

There's an unfortunate discrepancy between how we humans naturally interpret `const` due to its name (immutable value) vs. the compiler (immutable reference). I don't see that changing anytime soon.

If the ESLint rule ([prefer-const](https://eslint.org/docs/latest/rules/prefer-const)) had some sort of smart option like `["error", {"allowValueChange": false}]*/` then I'd use that option and keep `let` here. However, that's not possible and this rule has been enabled for all linted files so far, for example:

https://github.com/nextstrain/nextstrain.org/blob/adae78a29e34a38aba9bc91bf33d32ba0070976f/scripts/collect-datasets.js#L59-L63

So I plan to keep this change as-is for consistency without changing/ignoring ESLint rules.

Reply:
> So I plan to keep this change as-is for consistency without changing/ignoring ESLint rules.

Yep, absolutely the right choice for now.

> Yeah, `npm run lint` is configured to [run ESLint on files ending with `.js` and `.jsx`](https://github.com/nextstrain/nextstrain.org/blob/adae78a29e34a38aba9bc91bf33d32ba0070976f/package.json#L16).
> 
> This was an unintentional side effect of [cf0d1bf](https://github.com/nextstrain/nextstrain.org/commit/cf0d1bf430c85081de06e5fc1acdd86a4e159a46), but I think it's good to lint these files anyways.

Yeah, I'd put that together… but was/am confused because I think those scripts had `eslint-ignore` comments in them long prior to the change, and I don't think I would have written those comments had I not been badgered by eslint... so how'd they get checked previously? This is a rhetorical question; don't expect an answer from you and I'm not even inclined to go dig up the answer myself!